CONTRIBUTING.md

```diff
@@ -0,0 +1,166 @@
+
+    anzahl_freudenspruenge: int = attr.ib(validator=attr.validators.instance_of(int))
+    """
+    Anzahl Freudensprünge beschreibt, wie oft der CEO des Stromkonzerns in die Luft gesprungen ist, als ich den Vertrag unterschrieben habe.
+    Dieser Wert sollte im Normalfall mindestens 5 sein.
+    """
+    # todo: write a validator for anzahl_freudensprunge to be >5 and raise a ValidationError otherwise
+    # we can help you with anything you might be missing or unable to implement.
+    # You don't need to be a perfect programmer to contribute to bo4e :)
+
+    #: Optionale Menge (Elektrische Energie oder Gas oder Wärme), die ich zum Lieferbeginn umsonst erhalte
+    freimenge: Optional[Menge] = attr.ib(
+        validator=attr.validators.optional(attr.validators.instance_of(Menge)), default=None
+    )
+
+
+class MeinBoSchema(GeschaeftsobjektSchema):
+    """
+    Schema for de-/serialization of :class:`MeinBo`
+    """
+
+    # you don't need to copy paste the docstrings to the schema
+
+    class_name = MeinBo
+    # this is used so that the Pyhton object created on deserialization/loading has the correct type
+
+    # required attributes
+    lieferbeginn = fields.DateTime()
+    # the camelCase is implemented via he data_key kwarg
+    anzahl_freudenspruenge = fields.Int(data_key="anzahlFreudenspruenge")
+
+    # optional attributes
+    # when nesting you need to reference the schema here (not Menge but MengeSchema!)
+    freimenge = fields.Nested(MengeSchema, load_default=None)
+
+```
+
+### Unittests
+
+Ideally provide unittests that show:
+
+- that the BO/COM can be instantiated
+  - with only the required attributes
+  - with all attributes
+- can be serialized and deserialized again
+  - with only the required attributes
+  - with all attributes
+
+Therefore copy one of the existing "roundtrip" tests, see f.e. `TestTarifeinschraenkung`.
+
+## Pull Request
+
+Open a Pull Request against the main/default branch of this repository.
+We'd appreciate if you allowed maintainer edits.
+
+## Release Workflow
+
+- Check with tox all tests and lintings: `tox`
+- Check with tox if the packaging works fine: `tox -e test_packaging`
+- Squash Merge all your changes you would like to have in the release into the main/default branch
+- Check that all Github actions for tests and linting do pass (should be automatically enforced for PRs against master)
+- Go to the repositorys right side bar and click on "[Draft a new release](https://github.com/Hochfrequenz/BO4E-python/releases/new)"
+- Write in the _Tag version_ field and in the _Release title_ your new version, i.e. `v0.0.6`
+- Add a describtion to the release (or just autogenerate the change log which will be fine for 95% of cases)
+- Publish the release
+
+There is a Github Action which gets triggered by a release event.
+It will run all default tests with tox. If they pass, it will take the tag title to replace the version information in the _setup.cfg_ file.
```

Comment:
```suggestion
It will run all default tests with tox.
If they pass, it will take the tag title to replace the version information in the _setup.cfg_ file.
```